app/src/main/java/com/example/poste/activities/FolderViewActivity.java

```diff
@@ -72,47 +72,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         settingsBut.setOnClickListener(view -> {
             Intent intent = new Intent(FolderViewActivity.this, EditFolderActivity_v2.class);
+            intent.putExtra("folderId", PosteApplication.getSelectedFolder().getId());
+            intent.putExtra("folderName", PosteApplication.getSelectedFolder().getTitle());
             startActivity(intent);
-        });
-
-        // Create listeners for the post buttons
-        openBut.setOnClickListener(view -> {
-            if (PosteApplication.getSelectedPost() != null) {
-                try {
-                    Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(PosteApplication.getSelectedPost().getUrl()));
-                    startActivity(browserIntent);
-                } catch (Exception e) {
-                    e.printStackTrace();
-                    Toast.makeText(FolderViewActivity.this, R.string.internal_error, Toast.LENGTH_LONG).show();
-                }
-            }
-        });
-
-        editBut.setOnClickListener(view -> {
-            if (PosteApplication.getSelectedPost() != null) {
-                Intent intent = new Intent(FolderViewActivity.this, EditPostActivity.class);
-                startActivity(intent);
-            }
-        });
-
-        deleteBut.setOnClickListener(view -> {
-            if (PosteApplication.getSelectedPost() != null) {
-                PostDeletionCallback postDeletionCallback = new PostDeletionCallback() {
-                    @Override
-                    public void onSuccess() {
-                        Toast.makeText(FolderViewActivity.this, R.string.post_deleted, Toast.LENGTH_LONG).show();
-                        Intent intent = new Intent(FolderViewActivity.this, FolderViewActivity.class);
-                        finish();
-                        startActivity(intent);
-                    }
-
-                    @Override
-                    public void onError(String errorMessage) {
-                        Toast.makeText(FolderViewActivity.this, errorMessage, Toast.LENGTH_LONG).show();
-                    }
-                };
-                User.getUser().deletePostFromServer(PosteApplication.getSelectedPost(), postDeletionCallback);
-            }
+            finish();
```

Comment:
It might be best not to call finish() here so when the user is done editing, they are returned to this folder -- thoughts?

Reply:
Good catch I can address that in my ( https://tree.taiga.io/project/dzemlin-poste/us/185?milestone=368325 )